shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -437,14 +437,14 @@ static bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
 }
 
 void FlutterPlatformViewsController::ClipViewSetMaskView(UIView* clipView) {
-  if (clipView.maskView) {
+  if (clipView.layer.mask) {
     return;
   }
   UIView* flutterView = flutter_view_.get();
   CGRect frame =
       CGRectMake(-clipView.frame.origin.x, -clipView.frame.origin.y,
                  CGRectGetWidth(flutterView.bounds), CGRectGetHeight(flutterView.bounds));
-  clipView.maskView = [mask_view_pool_.get() getMaskViewWithFrame:frame];
+  clipView.layer.mask = [mask_view_pool_.get() getMaskViewWithFrame:frame];
```

Comment:
This is obviously still a draft since I haven't renamed `mask_view_pool` to `mask_layer_pool` etc.